packages/server/tests/acceptance/htsPrecompile_v1.spec.ts

```diff
@@ -0,0 +1,149 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// external resources
+import { solidity } from 'ethereum-waffle';
+import chai, { expect } from 'chai';
+
+chai.use(solidity);
+
+import { AliasAccount } from '../clients/servicesClient';
+import { ethers } from 'ethers';
+import ERC20MockJson from '../contracts_v1/ERC20Mock.json';
+import BaseHTSJson from '../contracts_v1/BaseHTS.json';
+
+
+describe('HTS Precompile V1 Acceptance Tests', async function () {
+  this.timeout(240 * 1000); // 240 seconds
+  const { servicesNode, relay } = global;
+
+  const TX_SUCCESS_CODE = 22;
+
+  const accounts: AliasAccount[] = [];
+  let BaseHTSContractAddress;
+  let HTSTokenContractAddress;
+  let NftHTSTokenContractAddress;
+  let NftSerialNumber;
+  let HTSTokenContract;
+  let baseHTSContract;
+  let baseHTSContractOwner;
+  let baseHTSContractReceiverWalletFirst;
+  let baseHTSContractReceiverWalletSecond;
+  let HTSTokenWithCustomFeesContractAddress;
+
+  this.beforeAll(async () => {
+    accounts[0] = await servicesNode.createAliasAccount(60, relay.provider);
+    accounts[1] = await servicesNode.createAliasAccount(15, relay.provider);
+    accounts[2] = await servicesNode.createAliasAccount(15, relay.provider);
+
+    // alow mirror node a 2 full record stream write windows (2 sec) and a buffer to persist setup details
+    await new Promise(r => setTimeout(r, 5000));
+
+    BaseHTSContractAddress = await deployBaseHTSContract();
```

Comment:
since this is the v1 test class where we're only testing the older stuff we can split these up a bit.
More the appropriate create call into the test method.
This way the test methods are actually testing create and should there be a regression we clearly see which one failed.